src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -18,282 +21,295 @@ const intlWrapper = children => (
     {children}
   </IntlProvider>
 );
+let axiosMock;
 
-describe('ProctoredExamSettings field dependency tests', () => {
-  beforeEach(async () => {
-    auth.getAuthenticatedHttpClient = jest.fn(() => ({
-      get: async () => ({
-        data: {
-          proctored_exam_settings: {
-            enable_proctored_exams: true,
-            allow_proctoring_opt_out: false,
-            proctoring_provider: 'mockproc',
-            proctoring_escalation_email: 'test@example.com',
-            create_zendesk_tickets: true,
-          },
-          available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
-          course_start_date: '2070-01-01T00:00:00Z',
-        },
-      }),
-    }));
-
-    auth.getAuthenticatedUser = jest.fn(() => ({ userId: 3, administrator: true }));
-    await act(async () => render(intlWrapper(<IntlProctoredExamSettings {...defaultProps} />)));
-  });
-
+describe('ProctoredExamSettings', () => {
```

Comment:
Note!  If you review this file with whitespace changes turned off, it's much more readable!  I put this describe block around the whole thing which indented everything.  Only 1/3 the changes are non-whitespace.